tests/CMakeLists.txt

```diff
@@ -1,22 +1,27 @@
 enable_language(CXX)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-local-typedefs -Wno-unused-parameter")
+if(MSVC)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+else()
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-local-typedefs -Wno-unused-parameter")
+endif(MSVC)
 
+include_directories(${OPENSSL_INCLUDE_DIR})
```

Comment:
In rsatest and ecdsa test I had to include `applink.c` or openssl would not work at all. See https://www.openssl.org/docs/faq.html (search for applink). I can move the include to WIN32 though.